src/PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.hpp

```diff
@@ -0,0 +1,283 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <boost/preprocessor/arithmetic/dec.hpp>
+#include <boost/preprocessor/arithmetic/inc.hpp>
+#include <boost/preprocessor/control/expr_iif.hpp>
+#include <boost/preprocessor/list/adt.hpp>
+#include <boost/preprocessor/repetition/for.hpp>
+#include <boost/preprocessor/repetition/repeat.hpp>
+#include <boost/preprocessor/tuple/to_list.hpp>
+#include <limits>
+#include <pup.h>
+#include <vector>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"  // IWYU pragma: keep
+#include "Utilities/Math.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace EquationsOfState {
+
+/*!
+ * \ingroup EquationsOfStateGroup
+ * \brief An equation of state given by parametrized enthalpy
+ *
+ * This equation of state is determined as a function of \f$x =
+ * \ln(\rho/\rho_0)\f$ where \f$\rho\f$ is the rest mass density and
+ * \f$\rho_0\f$ is the provided reference density.
+ * The pseudo-enthalpy \f$h \equiv (p + rho  + e)/rho\f$
+ * is expanded as
```

Comment:
Sorry, missed this inconsistency when I approved: given the definition of "e" as the total energy density below, we have
h= (e+p)/rho
as e includes the rest mass energy. Standard notation for GR-Hydro would be
h = (rho+u+p)/rho
where u is the internal energy density. You can certainly use some other notation, but "e" is used below with the definition
e = rho*(1+epsilon) = rho+u

Reply:
Ah yeah of course my bad, I'll change that quick